### ISSUE TITLE
stopword settingsUpdate FST error

### ISSUE THREAD

**Issue by Magorx15** [meilisearch/meilisearch#3893]
**Describe the bug**
When i set the stopwords of an index with certain stopwords it gives an internal FST error.
Examples that work:
```
{
    "stopWords":
    [
        "akkor"
    ]
}
```
```
{
    "stopWords":
    [
        "akár"
    ]
}
```

Doesn't work:
```
{
    "stopWords":
    [
        "akkor",
        "akár"
    ]
}
```

The error that i am getting:
```
"error": {
    "message": "internal: FST error.",
    "code": "internal",
    "type": "internal",
    "link": "https://docs.meilisearch.com/errors#internal"
}
```

**To Reproduce**
```
docker run -p 7700:7700 getmeili/meilisearch:v1.3.0-rc.0
```
```
curl -X POST 'http://localhost:7700/indexes' -H 'Content-Type: application/json' --data-binary '{"uid": "test", "primaryKey": "id"}'
```
```
curl -X PATCH 'http://localhost:7700/indexes/test/settings' -H 'Content-Type: application/json' --data-binary '{"stopWords": ["akkor", "akár"]}'
```

**Meilisearch version:**
v1.3.0-rc.0

**Comment by gillian-meilisearch** [comment]
Hi @Magorx15, thanks so much for spotting this, much appreciated  🙌 we'll take a look into it.

**Comment by dureuill** [comment]
Hi :wave:

Reproduced. I have a fix, see #3904 

Thank you for reporting 😃
